### ISSUE TITLE
Creating string from selection

### ISSUE THREAD

**Issue by mostobriv** [Vector35/binaryninja-api#3409]
**What is the feature you'd like to have?**
It's pretty useful to create strings with concrete length and without null-terminator (for example in rust binaries or firmwares).  

**Are any alternative solutions acceptable?**
atm afaik it can be done by manually setting the type of global variable, but it would be pretty handy to make automatically just with selection.

**Additional Information:**
Guess solution is behavior like in IDA: if selection is empty and A (or `Make string`  action is called) is pressed then just create string with assumption about null-terminator, otherwise create string with length equal to length of current selection.

**Comment by psifertex** [comment]
This is blocked by (and probably a duplicate of) #1334

**Comment by fuzyll** [comment]
It's possible this could be resolved by #3428 in some cases.

**Comment by xusheng6** [comment]
This has been fixed a while ago, though I do not have a specific commit/build number

'A' now behaves like this -- if you selection a single char, it looks for a null-terminator and create a null-terminated string with it. If you select multiple chars, then it creates the string data variable exactly as-is
